compositor_render/src/wgpu/texture.rs

```diff
@@ -182,6 +189,15 @@ impl InputTexture {
                             error!("Invalid texture passed as an input: {err}")
                         }
                     }
+                    InputTextureState::Nv12WgpuTexture(texture) => {
+                        let texture = NV12TextureView::from_wgpu_texture(texture.as_ref());
+                        let bind_group = texture.new_bind_group(ctx, ctx.format.nv12_layout());
```

Comment:
what will happen if the passed texture is not nv12?

e.g. for rgba texture with do some validation inside fill_from_wgpu_texture

Reply:
ok, handled that